docker-compose.ci.yml

```diff
@@ -4,7 +4,7 @@ services:
     image: "openapparelregistry:${GIT_COMMIT:-latest}"
 
   terraform:
-    image: quay.io/azavea/terraform:0.11.11
+    image: quay.io/azavea/terraform:0.11.14
```

Comment:
When I was upgrading the AWS provider version, I thought I might as well do this too since it's a patch bump. We'll be in a good place to do a Terraform 0.12 upgrade now.